adam-core/src/test/scala/org/bdgenomics/adam/rdd/feature/FeatureRDDSuite.scala

```diff
@@ -586,7 +586,7 @@ class FeatureRDDSuite extends ADAMFunSuite with TypeCheckedTripleEquals {
     // we can't guarantee that we get exactly the number of partitions requested,
     // we get close though
     assert(jRdd.rdd.partitions.length === 1)
-    assert(jRdd0.rdd.partitions.length === 5)
+    assert(jRdd0.rdd.partitions.length === 4)
```

Comment:
The current way things are implemented is that you can have more partitions than contigs. Whether or not this is correct is up for debate. Some partitions may not have data. Regardless of whether we grant their request for more partitions than data, copartitioning can certainly result in empty partitions. I thought about the problem of requesting more partitions than contigs, and figured that this edge case may not be worth the cost (rdd.count) of the verification that the number of partitions does not exceed the number of records (i.e. every partition has at least one record). We have to handle empty partitions anyway because of copartitioning.